tasks/tiler/xyz.py

```diff
@@ -14,21 +14,38 @@
 import csv
 
 LOGGER = get_logger(__name__)
+GEONAME_COLUMN = 'geoname'
+
+Z_RANGES = [
```

Comment:
Same ranges for all countries? Is this enough or do we need to be more flexible?

Reply:
It's just for partitioning. Currently, it's more than enough, there's no good reason for more fine-grained partitioning. I chose it based on the worst case, and it doesn't harm simpler cases.